resources/js/core/turbolinks-reload.ts

```diff
@@ -22,7 +22,7 @@ export default class TurbolinksReload {
 
   load(src: string) {
     if (this.loaded.has(src) || this.loading.has(src)) {
-      return this.loading.get(src)?.[0];
+      return this.loading.get(src)?.[0] ?? Promise.resolve();
```

Comment:
wouldn't it be `Promise.reject()`?

Reply:
this indicates the file has been loaded successfully